frontend/src/metabase/actions/containers/ActionParametersInputForm/utils.ts

```diff
@@ -51,3 +51,8 @@ export const getInitialValues = (
     ]),
   );
 };
+
+function stripTZInfo(dateOrTimeString: string) {
```

Comment:
This is super nit but thoughts on adding a short unit test for this? Could just be a single case, but would would be helpful for readers to see what an example input and output for this function could be